flask_dance/contrib/twitch.py

```diff
@@ -0,0 +1,111 @@
+"""Blueprint for connecting to Twitch API."""
+
+from flask_dance.consumer import OAuth2ConsumerBlueprint
+from functools import partial
+from flask.globals import LocalProxy, _lookup_app_object
+import os
+
+from flask import _app_ctx_stack as stack
+
+
+__maintainer__ = "Kerry Hatcher <kerry.hatcher@protonmail.com>"
+
+
+class MissingConfigException(Exception):
+    """Exception raised when a required config is missing."""
+
+
+def make_twitch_blueprint(
+    client_id=None,
+    client_secret=None,
+    scope=None,
+    redirect_url=None,
+    redirect_to=None,
+    login_url=None,
+    base_url="https://api.twitch.tv/helix/",
+    authorization_url="https://id.twitch.tv/oauth2/authorize",
+    token_url="https://id.twitch.tv/oauth2/token",
+    token_url_params={"include_client_id": True},
```

Comment:
These seem like too many options. The whole point of using a pre-set configuration is so that people don't _have_ to make as many choices. If a developer wants to specify all these options, she should just import and use the `OAuth2ConsumerBlueprint` class directly, rather than importing and using this configuration.

You can safely assume that anyone calling this function wants to make a blueprint that is specific to Twitch, so please remove any options that could allow them to make it work for any other OAuth provider.

Reply:
Roger, I think I understand.

Reply:
These options are still here. Did you forget to remove them?